snap/snapcraft.yaml

```diff
@@ -91,7 +91,19 @@ parts:
       - libxshmfence1
     prime:
       - usr/lib
-      - usr/share/X11
+      - usr/share/X11/XErrorDB
+
+  gbm:
```

Comment:
I am not sure if I need the gbm changes, do I?

Reply:
You do, yes. This enables things without eglstream support to work.